code/datums/voice_of_god_command.dm

```diff
@@ -294,16 +290,18 @@ GLOBAL_LIST_INIT(voice_of_god_commands, init_voice_of_god_commands())
 /datum/voice_of_god_command/knock_knock/execute(list/listeners, mob/living/user, power_multiplier = 1, message)
 	var/iteration = 1
 	for(var/mob/living/target as anything in listeners)
-		addtimer(CALLBACK(target, /atom/movable/proc/say, "Who's there?"), 5 * iteration)
+		addtimer(CALLBACK(target, /atom/movable/proc/say, "Who's there?"), 0.5 SECONDS * iteration)
 		iteration++
 
 /// This command forces silicon listeners to state all their laws.
 /datum/voice_of_god_command/state_laws
 	trigger = "state\\s*(your)?\\s*laws"
 
 /datum/voice_of_god_command/state_laws/execute(list/listeners, mob/living/user, power_multiplier = 1, message)
+	var/iteration = 1
 	for(var/mob/living/silicon/target in listeners)
-		target.statelaws(force = 1)
+		addtimer(CALLBACK(target, /mob/living/silicon/proc/statelaws, TRUE), 0.5 SECONDS * iteration)
```

Comment:
Given that stating laws generally spans several messages, we'll probably want the delay per iteration a bit greater than half a second.